srml/staking/src/lib.rs

```diff
@@ -746,6 +746,9 @@ decl_module! {
 				return Err("controller already paired")
 			}
 
+			let value = value.min(stash_balance);
```

Comment:
Why did you flip the order?

`stash_balance` is not defined until the line after now.

Reply:
yeah - a little too quick :)